app/views/field_test/experiments/_experiments.html.erb

```diff
@@ -0,0 +1,118 @@
+<style>
+  details details { padding: 1em 0 0 1.5em; }
+</style>
+<% experiments.reverse_each do |experiment| %>
+  <h2><%= experiment.name %><% unless experiment.active? %> <small class="closed">Completed</small><% end %></h2>
+  <details<%= " open" if experiment.active? %>>
+    <summary>Experiment Details</summary>
+
+    <% if experiment.description %>
+      <p class="description"><%= experiment.description %></p>
+    <% end %>
+
+    <details<%= " open" if experiment.active? %>>
+      <summary>Goals for <%= experiment.name %></summary>
+      <% experiment.goals.each do |goal| %>
+        <% results = experiment.results(goal: goal) %>
+
+        <% if experiment.multiple_goals? %>
+          <h3><%= goal.titleize %></h3>
+        <% end %>
+
+        <table>
+          <thead>
+            <tr>
+              <th>Variant</th>
+              <th style="width: 20%;">Participants</th>
+              <th style="width: 20%;">Conversions</th>
+              <th style="width: 20%;">Conversion Rate</th>
+              <th style="width: 20%;">Prob Winning</th>
```

Comment:
Non-blocking nit-picky comment: I would prefer if this said "Probability of Winning" even though it stretches the text out a bit.